### PR TITLE
wifi as both upstream and downstream

### PR DESCRIPTION
**Pull request by georgejhunt**
I read #197 and realized that some of my earlier work had not been shared. As I remember I did see gateway function with wifi acting both as client and server. 
But this experimentation was using dnsmasq, and I used a script to blast config files via "here documents".
I'm not taking on ownership of this objective, but I'm pretty sure it is possible to integrate within our current codebase
I know rc.local is not what we want long term -- so there is still quite a bit of work
Not smoke tested on recent master (though rebased on it), and not really a candidate for inclusion into master.

### CODE REVIEW

**Comment by holta** [comment]
Implements #430 experimentally, so this can be refined for [IIAB 6.6](https://github.com/iiab/iiab/milestone/3) &mdash; Thanks @georgejhunt

File: roles/8-mgmt-tools/meta/main.yml [hunk 1]
```diff
@@ -8,3 +8,4 @@ dependencies:
    - { role: phpmyadmin, tags: ['services','phpmyadmin','tools'], when: phpmyadmin_install }
    - { role: awstats, tags: ['services','awstats','tools'], when: awstats_install }
    - { role: teamviewer, tags: ['services','teamviewer','tools'], when: teamviewer_install }
+    - { role: network, tags: ['services','base','network'], when: gui_desired_network_role is defined and network_late }
```
Comment (thotmx) [inline roles/8-mgmt-tools/meta/main.yml:11]: Extra empty space breaks Travis CI

Comment (holta) [reply roles/8-mgmt-tools/meta/main.yml:11]: @georgejhunt please advise, Thanks!

**Comment by holta** [comment]
@georgejhunt should this be closed alongside #697 ?

(Or keep it open if there's good reason!)

**Pull request closed without merging**